web/client/epics/widgets.js

```diff
@@ -122,11 +122,13 @@ module.exports = {
 
     clearWidgetsOnLocationChange: (action$, {getState = () => {}} = {}) =>
         action$.ofType(MAP_CONFIG_LOADED).switchMap( () => {
-            const location = get(getState(), "routing.location");
+            const location = get(getState(), "routing.location").pathname.split('/');
```

Comment:
this can be moved in the widgets selectors with a test

Reply:
Good check, I'm pretty sure that there is already a selector for that. @baloola please use this:
https://github.com/geosolutions-it/MapStore2/blob/22bbfe82be2db71cd622d27bed457ccd41bb1332/web/client/selectors/routing.js#L5